### ISSUE TITLE
bugs1

### ISSUE THREAD

**Issue by DigiCripto** [desktop/desktop#4858]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I have an issue when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen. Add any considered drawbacks.

**Describe alternatives you've considered**

**Teachability, Documentation, Adoption, Migration Strategy**
If you can, explain how users will be able to use this. This will help the core team understand how the feature would work, and assist with future work like documentation.
Screenshots or mockups here are a great help!

**Comment by DigiCripto** [comment]
o

**Comment by agisilaos** [comment]
@DigiCripto thanks for reaching out!

We require the template to be filled out on all new issues. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

Thanks for understanding and meeting us halfway 😀

**Comment by caroladee** [comment]
Yes I apologize I honestly don't even know for sure what I'm doing I really
was just playing around trying things out but no clue for sure so again my
apologies. Apparently for now on I'll be a little more mindful of this
thank u
